src/Messenger/GenerateThumbnailsMessage.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Messenger;
+
+/**
+ * @author Jordi Sala Morales <jordism91@gmail.com>
```

Comment:
You can add `@psalm-pure` annotation here.

Reply:
It communicates that the object has no more dynamic parts / changing properties once the constructor was called:

https://psalm.dev/articles/immutability-and-beyond#purity

Reply:
Applied